sno/dataset2.py

```diff
@@ -541,12 +562,21 @@ def import_iter_meta_blobs(self, repo, source):
         schema = source.get_v2_schema()
         meta_blobs = [
             (self.VERSION_PATH, self.VERSION_IMPORT),
+            (self.TITLE_PATH, source.get_meta_item("title")),
+            (self.DESCRIPTION_PATH, source.get_meta_item("description")),
             self.encode_schema(schema),
             self.encode_legend(schema.legend),
         ]
 
+        # TODO - tidy up SRS ID code.
+        for srs in source.get_meta_item("gpkg_spatial_ref_sys"):
+            meta_blobs.append(
+                (f"{self.SRS_PATH}EPSG:{srs['srs_id']}.wkt", srs["definition"])
+            )
+
         for meta_path, meta_content in meta_blobs:
-            yield self.repo_path(meta_path), _bytes(meta_content)
+            if meta_content:
```

Comment:
is this correct? Means meta blobs with empty-string values won't be imported, which sounds incorrect. e.g. `description` is commonly an empty string.

Reply:
Good catch. Changed to "is not None" - that way, empty string / empty array is preserved as different to missing.